README.md

```diff
@@ -9,7 +9,7 @@ The fast mamba solver, now in conda!
 
 ## What is this exactly?
 
-conda-libmamba-solver is a new (experimental) solver for the
+conda-libmamba-solver is a new solver for the
```

Comment:
Dropped "experimental", but left "new". Idk if we want to drop both. Please let me know what you think :)

Reply:
Nah, this is fine, thank you!